### ISSUE TITLE
docs on using with Flask should mention jinja-assets-compressor

### ISSUE THREAD

**Issue by alanhamlett** [dahlia/libsass-python#84]
https://github.com/jaysonsantos/jinja-assets-compressor

**Comment by asottile** [comment]
Could you elaborate on this?  From what I can tell jinja-assets-compressor isn't a necessary component for using libsass-python with flask?

**Comment by alanhamlett** [comment]
The [Using with Flask Example](http://hongminhee.org/libsass-python/frameworks/flask.html#using-with-flask) should mention [jac](https://github.com/jaysonsantos/jinja-assets-compressor) as an alternative to SassMiddleware.

**Comment by asottile** [comment]
I'm not sure that's valid.  Our package provides SassMiddleware and not jac

**Comment by dahlia** [comment]
We could add a short mention to JAC, but the docs should still be mainly about the built-in `SassMiddleware`, I think.

**Comment by alanhamlett** [comment]
Yes, just wanted to mention JAC so people know the options available.
